subt_ign/launch/cloudsim_sim.ign

```diff
@@ -726,11 +736,21 @@
             <publish_sensor_pose>true</publish_sensor_pose>
             <publish_collision_pose>false</publish_collision_pose>
             <publish_visual_pose>false</publish_visual_pose>
-            <publish_nested_model_pose>#{$enableGroundTruth}</publish_nested_model_pose>
+            <publish_nested_model_pose>false</publish_nested_model_pose>
             <use_pose_vector_msg>true</use_pose_vector_msg>
             <static_publisher>true</static_publisher>
             <static_update_frequency>1</static_update_frequency>
           </plugin>
+          <plugin filename="libignition-gazebo-pose-publisher-system.so"
+            name="ignition::gazebo::systems::PosePublisher">
+            <publish_link_pose>false</publish_link_pose>
+            <publish_sensor_pose>false</publish_sensor_pose>
+            <publish_collision_pose>false</publish_collision_pose>
+            <publish_visual_pose>false</publish_visual_pose>
+            <publish_nested_model_pose>#{$enableGroundTruth}</publish_nested_model_pose>
+            <use_pose_vector_msg>true</use_pose_vector_msg>
+            <static_publisher>false</static_publisher>
+          </plugin>
```

Comment:
There are 4 pose-publishers in this file (one for `X1`, `X2`, `X3`, and `X4`), but it looks like you've only updated 2 of them here. Would you mind updating the other 2?

Reply:
Good catch, updated.